keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -350,6 +525,7 @@ does not compress the checkpoint archive on disk.
 * 2022-01-20: Reworked based on review and renamed feature gate to `ContainerCheckpoint`
 * 2022-04-05: Added CRI API section and targeted 1.25
 * 2022-05-17: Remove *restore* RPC from the CRI API
+* 2023-10-09: Beta graduation in 1.30
```

Comment:
nit: Can remove this and add it once we land the beta.

Reply:
removed